### ISSUE TITLE
Estonian admin hierarchy appears to be incorrect

### ISSUE THREAD

**Issue by ianthetechie** [whosonfirst-data/whosonfirst-data#2143]
This is bizarrely similar to #2141, so I'll keep it short. It looks like basically the entire city of Tallinn is a child of the localadmin for Mustamäe, one of the many local neighborhoods(?). I don't live there personally but am familiar enough to know that the hierarchy is incorrect.

Given the similarity to #2141, I wonder if there's some sort of common thread in how these were imported. It seems somewhat unlikely that two unrelated data sources so far apart had such a similar error involving grouping a significant number of neighborhoods under a single administrative unit.

Spelunker link: https://spelunker.whosonfirst.org/id/1713305847/descendants/?exclude=nullisland

**Comment by nvkelso** [comment]
From what I've seen, this type of problem can arises when source data imported to WOF is released by placetype, but local variation in administrative structures isn't easy to represent there by the national mapping agency and WOF perpetuates that confusion on import. 

We've slowly been cleaning these up, and it looks like you found 2 new examples we should fix by curating controlled hierarchies (with some other related placetype changes). The WOF PIP software will then respect those controlled hierarchies when otherwise performing automated point-in-polygon (PIP) admin hierarchy resolution lookups.

Take for example the case of New York City. Originally there were 5 counties, then those were consolidated to form the entire city, but legally there are still 5 counties (even if NYC calls them boroughs), so the US Census still includes them as 5 counties, even though locals think of them as subdivisions of the city instead. This problem is exacerbated when there is also `localadmin` placetype level in addition to `county` placetype level.

The upside is the problem is generally limited to capital cities just a handful of other very large cities (San Francisco comes to mind in the USA where it's entirely coterminious between city and county).

Please keep reporting problems like this, very useful feedback. Thank you.

**Comment by ianthetechie** [comment]
> From what I've seen, this type of problem can arises when source data imported to WOF is released by placetype, but local variation in administrative structures isn't easy to represent there by the national mapping agency and WOF perpetuates that confusion on import.

Makes sense...If it helps, here's my pseudo-local knowledge... Estonia's first layer of administrative hierarchy translates into English as a county, but this is probably represented as a region into WOF (since the first level is usually a region). That may have caused some confusion. Below that, at a "logical" level I believe cities/villages/etc. (localities in WOF taxonomy) like Tallinn, Tartu, Pärnu, and Narva come next. Mustamäe is a subdivision of Tallinn. I believe it is correctly classified as a localadmin per [Wikipedia](https://en.wikipedia.org/wiki/Tallinn#Administrative_districts). There are also non-administrative neighborhoods like Kalamaja, Noblessner, etc.

> The upside is the problem is generally limited to capital cities just a handful of other very large cities (San Francisco comes to mind in the USA where it's entirely coterminious between city and county).

Makes sense. Evidently I have an inconvenient knack for finding these ;)

> Please keep reporting problems like this, very useful feedback. Thank you.

Will do. Let me know if there's anything I can do to help fix this one as well (either personally or by connecting with a local).

**Comment by justinelliotmeyers** [comment]
https://geoportaal.maaamet.ee/eng/Spatial-Data/Administrative-and-Settlement-Division-p312.html
